core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/compare/CompareSetResult.java

```diff
@@ -0,0 +1,35 @@
+/*******************************************************************************
+ * Copyright (c) 2020 B2i Healthcare. All rights reserved.
```

Comment:
⚠️ Incorrect license. Please use the Apache 2.0 license.

Reply:
Done